### PR TITLE
Add Rappi case study

### PR DESCRIPTION
**Pull request by pymander**
This update adds the Rappi case study with reviewed/approved copy, and
a few assets to go with it.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by craigbox** [comment]
https://deploy-preview-10733--preliminary-istio.netlify.app/latest/about/case-studies/rappi/

**Review by craigbox — CHANGES_REQUESTED** [review]

File: content/en/about/case-studies/rappi/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+companyName: "Rappi"
+companyURL: "https://www.rappi.com/"
+logo: "/logos/rappi.png"
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 30
```
Comment (ericvn) [inline content/en/about/case-studies/rappi/index.md:17]: @craigbox Again, a question on weighting. If we keep increasing the number, then the new ones will be at the bottom, potentially requiring a bunch of scrolling.

File: .spelling [hunk 2]
```diff
@@ -694,13 +697,15 @@ quo
...
 Rajagopalan
+Rappi
 ratelimit-handler
 RawVM
 rbac
 re-applied
 re-patch
 reachability
 rearchitect
+rebalance
```
[the remark this answers is not in the document]
Comment (pymander) [reply .spelling:708]: @craigbox I did some googling and it looks like "rebalance" is a word.

File: content/en/about/case-studies/rappi/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The DevOps team behind Istio adoption is still moving forward with infrastructure improvements.
+
+"In the future, we hope to be able to implement multi-cluster support at the mesh level," says Arielli. "With this feature, it won't matter where an application is running. All applications will be able to access each other across clusters."
+
+## Conclusion: Successful Scaling with Istio
+
+Thanks to Istio, Rappi has managed to grow as the market requires. They can easily handle deployment of new clusters and turning on services in more markets and cities.
+
+"Thanks to Istio, we are confident that Rappi will be able to deploy more services and more applications to meet growing demand," says Arielli.
```
[the remark this answers is not in the document]
Comment (pymander) [reply content/en/about/case-studies/rappi/index.md]: Good catch. Fixed.

File: content/en/about/case-studies/rappi/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 30
+---
+
+![1500 deployments per day, 50+ clusters, 20,000 containers](./rappi-statistics.png)
```
[the remark this answers is not in the document]
Comment (pymander) [reply content/en/about/case-studies/rappi/index.md]: Done!

**Comment by pymander** [comment]
@craigbox Rebased and squashed!

File: content/en/about/case-studies/rappi/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Rappi Manages Growing Pains with Istio"
+linkTitle: "Rappi Manages Growing Pains with Istio"
+quote: "Istio seamlessly gave our team the ability to monitor services at any scale." 
+author:
+    name: "Ezequiel Arielli"
+    image: "/img/authors/ezequiel-arielli.jpg"
```
[the remark this answers is not in the document]
Comment (pymander) [reply content/en/about/case-studies/rappi/index.md:7]: New headshot has been added! I have squashed and rebased the branch. Should be ready.

File: content/en/about/case-studies/rappi/index.md (hunk 1 above)
Comment (craigbox) [reply content/en/about/case-studies/rappi/index.md:17]: @pymander 52 please

Comment (pymander) [reply content/en/about/case-studies/rappi/index.md]: @craigbox Done!

**Review by craigbox — APPROVED** [review]

**Pull request merged**